src/main/java/org/terasology/minimap/rendering/nui/layers/MinimapGrid.java

```diff
@@ -81,7 +79,7 @@
     private Texture textureAtlas;
     private TextureRegion questionMark;
 
-    private Multimap<BaseVector2i, Vector3i> dirtyBlocks = LinkedHashMultimap.create();
+    private Multimap<Vector2i, Vector3i> dirtyBlocks = LinkedHashMultimap.create();
```

Comment:
why not `Vector2ic`

Reply:
I guess that could work well. as long as the hashes are the same it should be fine even from the base type if joml ever adds an immutable vector. :+1: